docs/getting-started-scalardl-auditor.md

```diff
@@ -0,0 +1,545 @@
+# Getting Started with Helm Charts (Scalar DL Auditor)
```

Comment:
I left comments on the other guide, so please reflect the changes to this doc as well!